server/odc-service/src/main/java/com/oceanbase/odc/service/alarm/CloudTaskAlarmClient.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.alarm;
+
+import com.oceanbase.odc.service.alarm.model.CloudTaskAlarmEvent;
+
+/**
+ * @Author：tinker
+ * @Date: 2024/6/26 21:05
+ * @Descripition:
+ */
+public interface CloudTaskAlarmClient {
```

Comment:
we may add generic hook point in odc, then implements in odc cloud.

how about extends current AlarmService ?